xray/core/dataarray.py

```diff
@@ -912,6 +913,23 @@ def to_series(self):
         index = self.coords.to_index()
         return pd.Series(self.values.reshape(-1), index=index, name=self.name)
 
+    def to_masked_array(self, copy=True):
+        """Convert this array into a numpy.ma.MaskedArray
+
+        Parameters
+        ----------
+        copy : bool
+            If True (default) make a copy of the array in the result. If False
+            modify array in place and return a view.
```

Comment:
"Modify array in place" does not sound entirely accurate to me -- the array on the xray object is not modified